src/pages/commits/CommitsChart.tsx

```diff
@@ -29,12 +29,20 @@ interface Props {
   versions?: Commits;
   hasTaskFilter?: boolean;
   hasError?: boolean;
+  isOpenChartToggle: boolean;
```

Comment:
nit. `isChartOpen`

Reply:
I actually prefer to read `isOpenChart` as opposed to `isChartOpen`. Since it describes what the data represents and then what it belongs to.